compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -589,6 +587,8 @@ void addCPULinalgExtTileAndVectorizePipeline(
   // TODO: Should only apply decomposition here?
   funcPassManager.addPass(
       IREE::LinalgExt::createTileAndDecomposeAttentionPass());
+  funcPassManager.addPass(
+      IREE::LinalgExt::createDecomposeWinogradTransformPass());
```

Comment:
Is this not doing the tiling anymore?

Reply:
Yes, the tiling is all done in the tiled implementation now, so this pass just does decomposition.

Reply:
Nice! I see it now. (just I am dancing in my head cause this was what I have wanted for sooo long)

Reply:
Awesome! Thank you for making this happen :)